### ISSUE TITLE
repo2docker not working

### ISSUE THREAD

**Issue by bendichter** [catalystneuro/HDF5Zarr#10]
```
In file included from /tmp/pip-install-pv64tvc8/h5py/h5py/defs.c:656:0:
    /tmp/pip-install-pv64tvc8/h5py/h5py/api_compat.h:27:10: fatal error: hdf5.h: No such file or directory
     #include "hdf5.h"
              ^~~~~~~~
    compilation terminated.
```

see #8 

This comment: https://github.com/PacificBiosciences/pbcore/issues/5#issuecomment-76657454

suggests something like 

```
export HDF5_DIR=/path/to/hdf5
```

in the postBuild might do the trick

**Comment by bendichter** [comment]
or

```
$ python setup.py configure --hdf5=/path/to/hdf5
```

**Comment by d-sot** [comment]
Hello, repo2docker uses an environment variable `NB_PYTHON_PREFIX=${CONDA_DIR}/envs/notebook`. hdf5 installed with conda is in the same location. so instead of `CONDA_PREFIX`, using `NB_PYTHON_PREFIX` in the postBuild file works. Alternatively `HDF5_DIR=$(conda info --base)` could work as well.

**Comment by bendichter** [comment]
@arokem fixed by #11
